### PR TITLE
cli: Create schema 'views' if it does not exist

### PR DESCRIPTION
**Pull request by Bjwebb**

### CODE REVIEW

**Comment by jpmckinney** [comment]
Note that this is not in the code to reduce the required PostgreSQL privileges of the Kingfisher Summarize user. Please add a try/except to log an error that the schema needs to be created as documented.

**Comment by Bjwebb** [comment]
Doesn't Kingfisher Summarize need the same privileges to create the view_data_ schemas? https://github.com/open-contracting/kingfisher-summarize/blob/master/manage.py#L229-L231

**Comment by jpmckinney** [comment]
Indeed! I'm confused as to why this was ever a manual step.

Anyway, please update the docs and ci.yml, since this step is no longer necessary with this PR.

**Comment by Bjwebb** [comment]
I've updated the docs and ci.yml, as suggested.

**Comment by jpmckinney** [comment]
Thanks. Please avoid force-pushing as it makes it impossible to see new changes. If you are doing it to have a clean git log, then please just use "Squash and merge" after approval.

File: docs/get-started.rst [hunk 1]
```diff
@@ -69,20 +69,6 @@ Logging from the :doc:`cli/index` can be configured with a ``logging.json`` file
...
-
-#. `Create <https://www.postgresql.org/docs/current/sql-createschema.html>`__ the ``views`` schema, and set it to be owned by the database user configured above. For example, run:
-
-   .. code-block:: sql
-
-      CREATE SCHEMA views AUTHORIZATION ocdskingfisher;
-
-#. Close your PostgreSQL session, e.g. with ``Ctrl-D`` for both
-
 #. Create Kingfisher Summarize's configuration tables using the :ref:`install` command:
```
Comment (jpmckinney) [inline docs/get-started.rst:72]: No point in having a numbered list with a single item.

```suggestion
Create Kingfisher Summarize's configuration tables using the :ref:`install` command:
```

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**